.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+   extends: [
+      '@ni/eslint-config/typescript',
+      '@ni/eslint-config/typescript-requiring-type-checking',
+      'plugin:prettier/recommended'
+   ],
+   parserOptions: {
+      project: 'tsconfig.json'
+   },
+   rules: {
+      'no-console': 'off',
+      'import/no-extraneous-dependencies': 'off',
```

Comment:
This is mainly a problem inside the tests. They claim that modules like Mocha and other test utils should be defined as `dependencies` instead of `devDependencies`. But tests are only used in dev...

Reply:
Maybe we can configure the rule to identify tests correctly, something like

`'import/no-extraneous-dependencies': ['error', {'devDependencies': ['src/test/**/*.ts']}]`

Reply:
Nice solution, I like that 👍